control-base/src/main/java/fi/nls/oskari/control/data/GetPrintHandler.java

```diff
@@ -95,16 +96,16 @@ public GetPrintHandler(OskariLayerService layerService,
     public void handleAction(ActionParameters params) throws ActionException {
         PrintRequest pr = createPrintRequest(params);
         switch (pr.getFormat()) {
-        case PDF:
-            handlePDF(pr, params);
-            break;
-        case PNG:
-            handlePNG(pr, params);
-            break;
-        default:
-            throw new ActionParamsException(String.format(
-                    "Invalid value for key '%s'. Allowed values are: %s",
-                    PARM_FORMAT, ALLOWED_FORMATS));
+            case PDF:
```

Comment:
These indent-only changes inside the switch blocks aren't necessary, please remove them.